src/controllers/tripController.js

```diff
@@ -2,6 +2,7 @@ import * as tripServices from '../services/tripServices';
 import { respondWithSuccess, respondWithWarning } from '../helpers/responseHandler';
 import statusCode from '../helpers/statusCode';
 
+const { created, internalServerError } = statusCode;
```

Comment:
let's not destructure `statusCode`. let's reference the status codes directly with a default import of `statusCode`

Reply:
Makes sense